tests/unit/client.py

```diff
@@ -0,0 +1,442 @@
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied.  See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Unit tests for client library"""
+import haas
+from haas import model, api, deferred, server, config
+from haas.model import db
+from haas.test_common import *
+from haas.network_allocator import get_network_allocator
+import pytest
+import json
+
+import requests
+import os
+import tempfile
+import subprocess
+from subprocess import check_call, Popen
+from urlparse import urljoin
+import requests
+from requests.exceptions import ConnectionError
+# Hook to the client library
```

Comment:
I'm not sure what this comment is supposed to mean or why it's here. Same goes for the one in the cli.